src/components/network/network-explorer.js

```diff
@@ -80,8 +81,33 @@ const useStyles = makeStyles((theme) => ({
             color: theme.palette.type === 'dark' ? '#000' : '#FFF',
         },
     },
+    selectedTooltipName: {
+        height: (itemSize * 3) / 4,
```

Comment:
Do we really need this ad hoc css code ?
I guess the 230 is based on the drawer width (300), at least it should use the variable to compute the 230 from the 300